component.css

```diff
@@ -0,0 +1,549 @@
+:root{
+    --primary:#007bff;
+    --secondary:#1f2937;
+    --secondary-hover:#5b7280;
+    --primaryback: #cce5ff;
+    --primarytext: #004085;
+    --success:#5cb85c;
+    --successback: #d4edda;
+    --successtext: #155724;
+    --danger:#ff0000;
+    --dangerback: #f8d7da;
+    --dangertext: #721c24;
+    --warningback:#fff3cd;
+    --warningtext:#856404;
+    --grey-text:grey;
+}
+
+                                         /* Alerts  */
+.alert{
+    padding: 10px 5px;
+    border-radius: 7px;
+    text-align: center;
+    margin: 10px;
+}
+.alert-dismiss{
+    display: flex;
+    justify-content: center;
+}
+.dismiss{
+    border: none;
+    font-size: 20px;
+    outline: none;
+    position: absolute;
+    right: 55px;
+    cursor: pointer;
+    background-color: transparent;
+}
+.dismiss-primary{
+    color:var(--primarytext);
+}
+.dismiss-success{
+    color:var(--successtext);
+}
+.dismiss-danger{
+    color:var(--dangertext);
+}
+.dismiss-warning{
+    color:var(--warningtext);
+}
+.alert-primary{
+    background-color: var(--primaryback);
+    color:var(--primarytext);
+    border:1px solid var(--primarytext);
+}
+.alert-success{
+    background-color: var(--successback);
+    color:var(--successtext);
+    border:1px solid var(--successtext);
+}
+.alert-danger{
+    background-color: var(--dangerback);
+    color:var(--dangertext);
+    border:1px solid var(--dangertext);
+}
+.alert-warning{
+    background-color: var(--warningback);
+    color:var(--warningtext);
+    border:1px solid var(--warningtext);
+}
+
+                                              /* Avatar */
+.avatar-small{
+    width: 50px;
+    height: 50px;
+    border-radius: 50%;
+    border:2px solid blue;
+}
+.avatar-large{
+    width: 80px;
+    height: 80px;
+    border-radius: 50%;
+    border:2px solid blue;
+}
+.avatar-badge-time{
+    font-size: 10px;
+    background-color:  green;
+    color: white;
+    padding: 1px 6px;
+    border: 2px solid white;
+    border-radius: 10px;
+    position: relative;
+    left: -16px;
+    top: -4px;
+}
+.avatar-badge-active{
+    font-size: 10px;
+    background-color:  green;
+    color: white;
+    padding: 1px 6px;
+    border: 2px solid white;
+    border-radius: 50%;
+    position: relative;
+    left: -16px;
+    top: -4px;
+}
+.avatar-text{
+    padding: 11px 18px;
+    border: 1px solid var(--primary);
+    background-color: black;
+    border-radius: 50%;
+    width: fit-content;
+    height: fit-content;
+}
+.avatar-text > span{
```

Comment:
try to create a separate class for this type of class(I don't remeber the name, XD). This way some other class may take preference and mess up your design